keyboards/ropro/config.h

```diff
@@ -0,0 +1,35 @@
+#pragma once 
+
+#include "config_common.h" 
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0002
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Kingly_keys
```

Comment:
Just my opinion, but I think this would look better.

```suggestion
#define MANUFACTURER    Kingly-Keys
```

Reply:
IIRC the preprocessor sometimes has issues with hyphens in places like this. Might be alright, but just something to watch out for.

Reply:
If that's the case, I would think wrapping the value in quotation marks (`"Kingly-Keys"`) should fix it.